### ISSUE TITLE
Unnecessary warnings due to inline assembly stabilized in 1.59

### ISSUE THREAD

**Issue by Programatic** [rust-osdev/bootloader#226]
Hello,

I am getting the following warnings originating from bootloader. As the warning says, in the latest versions inline assembly has been stabilized and thus these feature gates are causing unnecessary clutter in the terminal.

```rust
warning: the feature `asm` has been stable since 1.59.0 and no longer requires an attribute to enable
 --> src/main.rs:3:12
  |
3 | #![feature(asm)]
  |            ^^^
  |
  = note: `#[warn(stable_features)]` on by default

warning: the feature `global_asm` has been stable since 1.59.0 and no longer requires an attribute to enable
 --> src/main.rs:2:12
  |
2 | #![feature(global_asm)]
  |    
```

Edit: I kept confusing the names, sorry :/

**Comment by phil-opp** [comment]
Thanks for reporting! 

Since you're using `bootimage`, I assume that you're using a `v0.9` release of the `bootloader` crate. This version does indeed still tries to enable the stabilized `asm` feature. I'll fix that.

For the `v0.10` release, this was fixed in https://github.com/rust-osdev/bootloader/pull/210, which was released in `bootloader` v0.10.11. Running `cargo update -p bootloader` should fix it.

**Comment by Programatic** [comment]
I was just following through the series you have on your website, so I assumed that it was not updated yet when I posted this issues, my bad! I tried using v0.10, but couldn't get it working in the few minutes I had spent on it. Is there plans to update the blog to use the newest version, or is this an area that needs help?

**Comment by phil-opp** [comment]
Yeah, the blog does not work with bootloader v0.10 yet. There were some substantial changes in v0.10 (e.g. UEFI support), so parts of the blog require a rewrite. However, I'm not fully satisfied with the build system of v0.10 yet, so I'm currently working on a new v0.11 version, which I will then use for the next iteration of the blog.

This means that things are in a weird state right now, where both the v0.9 and v0.10 versions of the bootloader crate are actively updated. Sorry for the confusion! I created a PR to fix these warnings in https://github.com/rust-osdev/bootloader/pull/227.

**Comment by phil-opp** [comment]
Fixed in #227
